pageserver/src/tenant/size.rs

```diff
@@ -40,6 +41,21 @@ struct TimelineInputs {
     next_gc_cutoff: Lsn,
 }
 
+/// Gathers the inputs for the tenant sizing model.
+///
+/// Tenant size does not consider the latest state, but only the state until next_gc_cutoff, which
```

Comment:
> until

Confused, since always thought that the size we track is after all GC cutoffs. Am I wrong?

Reply:
https://neondb.slack.com/archives/C043L4SG17B/p1666818946663559?thread_ts=1665597859.715069&cid=C043L4SG17B -- I am trying to get oldest LSN within the retention period and calling it next_gc_cutoff. Me calling it that doesn't make correctly calculated right now, need to ponder this as well.

Reply:
I've tried asking questions based on the new docs I've read with my basic understanding of Lsn as "incremental data measurement": Lsn can only increase for any change in the DB, so when I read "state until next GC cutoff" as some state that gets to be "cut off (i.e. deleted) by next time GC runs", so I might not understand the GC terms good enouth.

Reply:
I am not 100% either on this. However I like that it's not `gc_cutoff` for this reason of raising questions. So I will have to postpone answering the question :)